pytket/extensions/pyquil/pyquil_convert.py

```diff
@@ -71,11 +71,16 @@
     "PHASE": OpType.U1,
     "SWAP": OpType.SWAP,
     "XY": OpType.ISWAP,
+    "CH": OpType.CH,
+    "CY": OpType.CY,
 }
 
 
 _known_quil_gate_rev = {v: k for k, v in _known_quil_gate.items()}
 
+# Gates with single control
+_single_control_gates = ["CH", "CY"]
```

Comment:
Why is CNOT and CPhase not in this lost?

Reply:
And also `CZ`?

Reply:
`CNOT` would not work with the logic below because there is no `NOT` gate, I think.

Reply:
You could do X for Cnot?

Reply:
But line 311 assumes that the name for character 1 onwards is a valid gate name. This does seem like fragile assumption, and I think it would be better to implement this as a dictionary (name of controlled gate --> name of gate).